conn/physic/units.go

```diff
@@ -207,6 +209,23 @@ func (f Frequency) String() string {
 	return microAsString(int64(f)) + "Hz"
 }
 
+// Set sets the Frequency to the value represented by s. Units are to be
+// provided in hertz with optional SI prefixes.
+func (f *Frequency) Set(s string) error {
+	v, n, err := valueOfUnitString(s, micro)
+	if err != nil {
+		return err
+	}
+
+	switch strings.ToLower(s[n:]) {
+	case "hz", "hertz":
+		*f = (Frequency)(v)
+	default:
```

Comment:
for naked units we could do something like:
```golang
case "":
    *f = (Frequency)(v)
default:
    incorrectUnits(s[n:], "Hz" )
```

Reply:
I did this.

My thoughts are if the units are unambiguous, ie: no other units then its fine to allow naked units?

So frequency I think is fine, someone may want rpm but they can do the logic for that themselves.

Reply:
ok lets go with that then.